### PR TITLE
Added get methods for the UserStat class and Checkstyle

### PR DESCRIPTION
**Pull request by anestornahuel**

### CODE REVIEW

**Comment by ConCas27** [comment]
Issue number 118 was solved by castillo and alvarez on the same computer

File: src/main/java/unrc/dose/UserStat.java [hunk 1]
```diff
@@ -1,64 +1,102 @@
...
+ */
 public class UserStat extends Model {
-	/*Validators for model*/
-	static {
-		validatePresenceOf("user_id");
-		validatePresenceOf("created_challenges");
-		validatePresenceOf("solved_challenges");
-		validatePresenceOf("current_points");
-	}
+/* Validators for model */
```
Comment (pponzio) [inline src/main/java/unrc/dose/UserStat.java]: Please fix indentation issues.

Comment (pponzio) [reply src/main/java/unrc/dose/UserStat.java]: And run checkstyle before commiting again.

File: src/main/java/unrc/dose/UserStat.java [hunk 2]
```diff
@@ -1,64 +1,102 @@
...
+     * Create the statistics for a given user.
+     * @param id The id of the user who statistics we will create.
+     */
+    public static void createUserStat(final int id) {
+        UserStat stat = new UserStat();
+        stat.set("user_id", id);
+        stat.set("created_challenges", 0);
+        stat.set("solved_challenges", 0);
+        stat.set("current_points", 0);
+        Base.openTransaction();
```
Comment (pponzio) [inline src/main/java/unrc/dose/UserStat.java]: It is recommended to not use transactions here. Just stat.saveIt().

File: src/test/java/unrc/dose/UserStatTest.java [hunk 3]
```diff
@@ -76,29 +132,33 @@ public void getUserStat() {
...
+		u2.set("email_address", "LaMosquita@gmail.com");
+		u2.save();
+		UserStat.createUserStat(u2.getInteger("id"));
+		LazyList<UserStat> userStats = UserStat.showAllUserStat();
+		for (int i = 0; i < userStats.size(); i++){
+			UserStat stat = userStats.get(i);
+			assertTrue(stat.getUserId() == u.getId() || stat.getUserId() == u2.getId());
 		}
+    u.delete();
+    u2.delete();
```
Comment (pponzio) [inline src/test/java/unrc/dose/UserStatTest.java]: Indentation issues.

File: src/test/java/unrc/dose/UserStatTest.java [hunk 4]
```diff
@@ -24,21 +24,76 @@ public static void beforeAll() {
...
 	@AfterClass
 	public static void afterAll() {
 		log.info("UserStatTest AfterClass");
 		Base.close();
 	}
-	
-	@After
-	public void afterTest() {
-		Base.rollbackTransaction();
+
```
Comment (pponzio) [inline src/test/java/unrc/dose/UserStatTest.java]: For tests it is recommended to use DBSpec. Furthermore, a beforeClass methods should be defined, to populate the DB with the information relevant to the tests.

File: src/test/java/unrc/dose/UserStatTest.java [hunk 5]
```diff
@@ -24,21 +24,76 @@ public static void beforeAll() {
...
+	/**
+	 * Test the method getUserId from
+	 * UserStat class.
+	 */
+	@Test
+	public void getUserIdTest() {
+		User u = new User();
```
Comment (pponzio) [inline src/test/java/unrc/dose/UserStatTest.java]: Use the method 

public static User set(final String name, final String pass,
        final String email, final Boolean admin)

from the User class to create a new User.

**Pull request merged**